packages/nvidia_nat_strands/tests/test_strands_integration.py

```
@@ -0,0 +1,564 @@
+
+        async def calculator_impl(input_data: CalculatorInput) -> CalculatorOutput:
+            """A simple calculator that performs basic arithmetic operations."""
+            if input_data.operation == "add":
+                result = input_data.a + input_data.b
+            elif input_data.operation == "subtract":
+                result = input_data.a - input_data.b
+            elif input_data.operation == "multiply":
+                result = input_data.a * input_data.b
+            elif input_data.operation == "divide":
+                if input_data.b == 0:
+                    raise ValueError("Cannot divide by zero")
```

Ruff (0.14.5)

Avoid specifying long messages outside the exception class

(TRY003)